src/Paramore.Brighter.MessagingGateway.AzureServiceBus/AzureServiceBusMessageProducer.cs

```diff
@@ -248,21 +248,40 @@ private async Task<IServiceBusSenderWrapper> GetSenderAsync(string topic)
         }
     }
 
-    private static ServiceBusMessage ConvertToServiceBusMessage(Message message)
+    private static ServiceBusMessage ConvertToServiceBusMessage(Message message, AzureServiceBusPublication publication)
     {
-        var cloudEvent = new CloudEvent(message.Header.Source.ToString(),
-            message.Header.Type,
-            new BinaryData(message.Body.Value))
+        BinaryData payload;
+        if (publication.UseCloudEvents)
```

Comment:
Do we need this? The message mapper takes care of determining the header and body of the message. The publisher would not need to understand this.

Reply:
Make sense, I'll revert it